src/net/ICE/RtpIceChannel.cs

```diff
@@ -166,11 +166,11 @@ public List<RTCIceCandidate> Candidates
         {
             get
             {
-                return _candidates;
+                return _candidates.ToList();
             }
         }
 
-        private List<RTCIceCandidate> _candidates = new List<RTCIceCandidate>();
+        private ConcurrentBag<RTCIceCandidate> _candidates = new ConcurrentBag<RTCIceCandidate>();

```

Comment:
Is the switch from List to ConcurrentBag precautionary or did you run into a threading issue?